src/App.js

```diff
@@ -14,7 +14,7 @@ const App = () => {
   };
 
   return (
-    <>
+    <React.Fragment>
       <Header onCategoryChange={handleCategoryChange} />
       <div className="flex justify-content-center" style={{ marginTop: "50px", padding: '50px' }}>
```

Comment:
Nice good catch again, as I have just looked and ran on my local dev server, and it functions normally with `<div className="flex justify-content-center" >` with your other proposed change on `App.js l.21: there is margin and padding added on l.19 already —> can be removed (not changed)`